governance/polls/Budget Ratification Poll for Modify Growth Core Unit Budget, GRO-001 (MIP40c3-SP70) - June 13, 2022.md

```diff
@@ -0,0 +1,82 @@
+
+## Review
+
+The community may vote in this poll to express support or opposition to MIP40c3-SP70 being accepted and implemented in the Maker Protocol by ranking their support of the following options:
+* **Option 1:** Approve increased budget (BaseGrowth): 5,085,157.11 DAI.
+* **Option 2:** Approve increased budget (BearMarket): 4,210,261.40 DAI.
+* **Option 3:** Approve existing budget only (BrandlessGrowth): 3,272,550.00 DAI
+* **Option 4:** Reject budget.
+
+A brief summary of this proposal has been provided by the MIP Author and is shown below:
+
+*MIP40c3-SP70 requests a 12-month budget of 5,085,157.11 DAI for the Growth Core Unit starting from July 2022 (Option BaseGrowth).*
+
+*Due to the current market situation, we decided to propose a second option (option BearMarket) with a reduction in the objective FTEs, which will slow down our roadmap for H2-22 to H1-23. Option BearMarket requests a 12-month budget of 4,210,261.40 DAI for the Growth Core Unit starting from July 2022.*
+
+*Personnel reduction from the original plan, although feasible, will have an impact on the roadmap execution of both teams (Growth and Brand) and the number of strategic initiatives we can run with our partners. In case option BearMarket is elected, we will:*
+
+* *Reduce the number of new business opportunities or associations that require a high intensity of work (usually high impact partnerships).*
+* *Instead of having a specialization approach (per region and vertical), we will continue with a generalist approach.*
+* *Merge the communications and content team.*
+* *Instead of having a separate team working on the brand, BDs will continue developing the Maker brand with our partners, with the PR agencies' support.*
+
+*Although this is a more conservative approach and gives us time as a DAO to define the next steps towards MetaDAOs or any other path MKR holders decide, it's essential to understand that this will slow down Growth's CU roadmap (BDs+Brand).*
+
+**Option BaseGrowth** *(5,085,157.11 DAI) represents a 55% increase compared to last year's approved budget. This increase in budget will support our growth in FTE, allowing us to expand our presence in other regions and have all people needed to run the Brand team.*
+**Option BearMarket** *(4,210,261.40 DAI) represents a 28% increase compared to last year's approved budget. We won't be growing the team as planned but will be able to execute our plan with some constraints.*
+**Option BrandlessGrowth** *(3,272,550.00 DAI) represents a 0% increase compared to last year's approved budget. If MKR holders select his option, we won't be able to continue with the Brand team plan.*
+
+Please review the links below to inform your position on this proposal before voting.
+* [Canonical Proposal Version](https://github.com/makerdao/mips/blob/8f65911bd013055b7914ac141a7d4ee9ba9e61da/MIP40/MIP40c3-Subproposals/MIP40c3-SP70.md)
```

Comment:
Use this URL: https://github.com/makerdao/mips/blob/34cce188049e6d6d1adcfcaf0c4fde1c6e4f7085/MIP40/MIP40c3-Subproposals/MIP40c3-SP70.md